src/scene/forward-renderer.js

```diff
@@ -35,6 +35,12 @@ pc.extend(pc, function () {
         new pc.Mat4().setScale(0.5, 0.5, 0.5)
     );
 
+    var rgbaDepthClearOptions = {
+            color: [254.0/255, 254.0/255, 254.0/255, 254.0/255],

```

Comment:
We should consider some auto-formatting tools, because this is rather ridiculous